scripts/version.sh

```diff
@@ -29,7 +29,7 @@ fi
 diff=`git diff-index HEAD | wc -l`
 if [[ $diff -gt 0 ]]; then
     echo "There are modified files in your working copy (or staged in the index). Please commit or stash them and rerun this command."
-    exit 1
+    # exit 1
```

Comment:
Is this some debugging leftover? I'd really avoid leaving modified files in the tree while running this script.

Reply:
@laurentS I have undo the changes and pushed.